app/controllers/v0/profile/transactions_controller.rb

```diff
@@ -11,14 +11,17 @@ def status
           service,
           transaction_params[:transaction_id]
         )
+
         raise Common::Exceptions::RecordNotFound, transaction unless transaction
+        Vet360Redis::Cache.invalidate(@current_user)
```

Comment:
Do we want to do this every time this request is made? Or only when the transaction refresh contains updates?

Reply:
Yeah, I thought about that a bit, too.  IMO, we should error on the side of caution, and invalidate it every time.